### PR TITLE
[System Settings] Change valid_languages format 

### LINKED ISSUE #14372: [System Settings] Change valid_languages format
Currently the `valid_languages` is configured CSV: 

```yaml
pimcore:
    general:
        valid_languages: 'en,de,fr'
```

This was for legacy reasons and we should change that to proper  array format: 

```yaml
pimcore:
    general:
        valid_languages: 
            - en
            - de
            - fr
```

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request  
Resolves #14372

### CODE REVIEW

File: bundles/AdminBundle/config/pimcore/config.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
 imports:
   - { resource: default.yaml }
-  - { resource: two_factor_authentication.yaml }
\ No newline at end of file
+  - { resource: two_factor_authentication.yaml }
+
+monolog:
+    handlers:
+        error:
+            type: stream
```
Comment (brusch) [inline bundles/AdminBundle/config/pimcore/config.yaml]: This seems to be unrelated to this PR, right?

**Review by brusch — CHANGES_REQUESTED** [review]

File: bundles/AdminBundle/config/pimcore/config.yaml (hunk 1 above)
Comment (lukmzig) [reply bundles/AdminBundle/config/pimcore/config.yaml]: You are right, sorry for that. It is reverted now.

File: bundles/CoreBundle/src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -190,8 +190,15 @@ private function addGeneralNode(ArrayNodeDefinition $rootNode): void
                     ->defaultValue('en')
                 ->end()
                 ->arrayNode('valid_languages')
-                    ->prototype('scalar')->end()
+                    ->info("String or array format are supported.")
+                    ->beforeNormalization()
+                    ->ifString()
+                        ->then(function ($v) {
+                            return explode(',', $v);
+                        })
```
Comment (jdreesen) [inline bundles/CoreBundle/src/DependencyInjection/Configuration.php]:
```suggestion
                        ->then(fn ($v) => explode(',', $v))
```

Comment (lukmzig) [reply bundles/CoreBundle/src/DependencyInjection/Configuration.php]: Thank you, updated
